src/components/sub-components/mgt-flyout/mgt-flyout.ts

```diff
@@ -18,7 +19,7 @@ import { styles } from './mgt-flyout-css';
  * @extends {LitElement}
  */
 @customElement('mgt-flyout')
-export class MgtFlyout extends LitElement {
+export class MgtFlyout extends MgtBaseComponent {
```

Comment:
Is there a reason why MgtFlyout extends MgtBaseComponent? Unless there is a reason, we should stick with LitElement

Reply:
I switched it to get access to the `fireCustomEvent` method, but I can use dispatchEvent instead and go back to LitElement.

Reply:
Please do, no need to bring in everything from BaseComponent just for events